src/components/Header.jsx

```diff
@@ -11,6 +13,7 @@ const Header = () => {
         as={Link}
         to={{ pathname: "/" }}
         data-cy="home"
+        onClick={Article.index}
```

Comment:
Article.index happens not onClick but on initial render

Reply:
Now removed, using initial render instead.